From d3554ee25d68ba576fdc5a0044d45528e3a580a1 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Mon, 28 Jan 2019 13:46:42 -0800
Subject: [PATCH 02/10] Add -fopenmp

---
 FAKE | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 FAKE

diff --git a/FAKE b/FAKE
new file mode 100644
index 0000000..76688b7
--- /dev/null
+++ b/FAKE
@@ -0,0 +1 @@
+Placeholder for empty commits.
-- 
2.20.1

