From 76073111e68ed3745b239047cc9002bb652eafd9 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Wed, 24 Oct 2018 11:31:44 -0700
Subject: [PATCH 3/3] Enable python bindings to be built automatically.

---
 inference-engine/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/inference-engine/CMakeLists.txt b/inference-engine/CMakeLists.txt
index 2f561d9..5232b7d 100644
--- a/inference-engine/CMakeLists.txt
+++ b/inference-engine/CMakeLists.txt
@@ -127,6 +127,12 @@ endif(UNIX)
 add_subdirectory(src)
 add_subdirectory(tests)
 add_subdirectory(thirdparty)
+
+if (ENABLE_PYTHON_BINDINGS)
+    set(InferenceEngine_DIR "${CMAKE_BINARY_DIR}")
+    add_subdirectory(ie_bridges/python)
+endif()
+
 if (ENABLE_SAMPLES_CORE)
     set(InferenceEngine_DIR "${CMAKE_BINARY_DIR}")
 
-- 
2.19.1

