From b27eff48118823153c5ecdcb3082720739af482a Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Tue, 29 Jan 2019 11:04:44 -0800
Subject: [PATCH 10/16] Don't look for ade in a subdir

---
 inference-engine/thirdparty/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/inference-engine/thirdparty/CMakeLists.txt b/inference-engine/thirdparty/CMakeLists.txt
index f65f38c..c8b0bf9 100644
--- a/inference-engine/thirdparty/CMakeLists.txt
+++ b/inference-engine/thirdparty/CMakeLists.txt
@@ -11,7 +11,6 @@ add_subdirectory(pugixml)
 export(TARGETS pugixml NAMESPACE IE:: APPEND FILE "${CMAKE_BINARY_DIR}/targets.cmake")
 
 add_subdirectory(stb_lib)
-add_subdirectory(ade)
 
 if (ENABLE_CLDNN)
     set(CLDNN__OUTPUT_BIN_DIR ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
-- 
2.21.0

