--- dldt-2018_R3/inference-engine/src/inference_engine/CMakeLists.txt~	2018-10-16 11:00:32.000000000 +0000
+++ dldt-2018_R3/inference-engine/src/inference_engine/CMakeLists.txt	2018-10-24 13:58:31.491843624 +0000
@@ -95,4 +95,6 @@
 configure_file(
     "${CMAKE_SOURCE_DIR}/cmake/share/InferenceEngineConfig-version.cmake"
     "${CMAKE_BINARY_DIR}/InferenceEngineConfig-version.cmake"
-    COPYONLY)
\ No newline at end of file
+    COPYONLY)
+
+install(TARGETS ${TARGET_NAME} LIBRARY DESTINATION lib64)
\ No newline at end of file
--- dldt-2018_R3/inference-engine/src/inference_engine/CMakeLists.txt~	2018-10-24 13:58:31.000000000 +0000
+++ dldt-2018_R3/inference-engine/src/inference_engine/CMakeLists.txt	2018-10-24 15:15:48.045779070 +0000
@@ -61,7 +61,7 @@
 target_include_directories(${TARGET_NAME} SYSTEM PRIVATE "${IE_MAIN_SOURCE_DIR}/thirdparty/ade/ade/include")
 target_include_directories(${TARGET_NAME} SYSTEM PRIVATE "${IE_MAIN_SOURCE_DIR}/thirdparty/ade/common/include")
 
-set_target_properties(${TARGET_NAME} PROPERTIES COMPILE_PDB_NAME ${TARGET_NAME})
+set_target_properties(${TARGET_NAME} PROPERTIES COMPILE_PDB_NAME ${TARGET_NAME} SOVERSION 1)
 
 # Static library used for unit tests which are always built
 
@@ -81,7 +81,7 @@
 
 target_compile_definitions(${TARGET_NAME}_s PUBLIC -DUSE_STATIC_IE)
 
-set_target_properties(${TARGET_NAME}_s PROPERTIES COMPILE_PDB_NAME ${TARGET_NAME}_s)
+set_target_properties(${TARGET_NAME}_s PROPERTIES COMPILE_PDB_NAME ${TARGET_NAME}_s SOVERSION 1)
 
 # export targets
 
